src/Sylius/Bundle/ApiBundle/DependencyInjection/SyliusApiExtension.php

```diff
@@ -30,5 +31,11 @@ public function load(array $configs, ContainerBuilder $container): void
         $container->setParameter('sylius_api.product_image_prefix', $config['product_image_prefix']);
 
         $loader->load('services.xml');
+
+        if (!$container->hasParameter('enable_swagger') || !$container->getParameter('enable_swagger')) {
+            return;
+        }
+
+        $loader->load('swagger/swagger.xml');
```

Comment:
Almost all of our integrations are placed in `services/integrations/*.xml` files. Yet, placing it there forbids us from usage of import wildcards, so my proposal is to adjust path to:
```suggestion
        $loader->load('integartions/swagger.xml');
```

Reply:
@lchrusciel this is a special case because we shouldn't load it if swagger is turned off. So if we use wildcard here it could be loaded when it's off too and make the same bug.